client/core/notification.go

```diff
@@ -157,6 +157,14 @@ func (c *Core) formatDetails(topic Topic, args ...any) (translatedSubject, detai
 	return trans.subject, c.localePrinter.Sprintf(string(topic), args...)
 }
 
+func makeTxToken(txHash string, assetID uint32) string {
```

Comment:
`makeCoinIDToken`. It's not just a tx hash for utxo assets and its an address sometimes for evm assets.